### ISSUE TITLE
70001 internal error

### ISSUE THREAD

**Issue by ivan-volnov** [Tinkoff/investAPI#381]
Проверьте пожалуйста вот эти tracking_id:

6406fb5c00f4496b240f9a6d288136ed
6406e27c896659fa7433aff8dc4025fb
6406de9a3922afff5097253c6a3207dc
6406e2418d0bfa3022f98a90e4b3d7c2

**Comment by AlexanderVolkovTCS** [comment]
>6406fb5c00f4496b240f9a6d288136ed

"Ошибка отмены поручения"

>6406e27c896659fa7433aff8dc4025fb, 6406e2418d0bfa3022f98a90e4b3d7c2

"Не найдена заявка по id"

>6406de9a3922afff5097253c6a3207dc

внутренний таймаут вызова. В этом случае ошибка 70001 верная, по остальным случаям заявка уже видимо исполнилась или отменилась, но на это есть отдельные коды. Уточняю у коллег, почему вместо них 70001 пришла.

Спасибо за репорт.

**Comment by ivan-volnov** [comment]
> "Не найдена заявка по id"

Да, это видимо исправляется в #374. Все равно код ошибки мог бы быть более информативным нежели internal error.

> Уточняю у коллег, почему вместо них 70001 пришла.

Всегда приходит, кстати. Такое поведение не единичное.

Спасибо. Жду исправлений!
